regression/goto-diff/java-scope-change/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+new.jar
+old.jar --json-ui
+// Enable multi-line checking
+activate-multi-line-match
+EXIT=0
+SIGNAL=0
+  "deletedFunctions": \[ \],
+  "modifiedFunctions": \[\n    {\n      "name": "java::Test\.foo:\(I\)I",\n      "sourceLocation": {\n        "file": "Test\.java",\n        "function": "java::Test\.foo:\(I\)I",\n        "line": "4"\n      }\n    }\n  \],
```

Comment:
I'd expect both methods to be modified (both have become non-public). Is the modification in the body of foo intended?

Reply:
Thanks Peter, the modification in the body was a copy and paste error. I have fixed this with the last upload. I have also marked this test as KNOWNBUG given that it now fails.

You are correct that both methods should be marked as modified I have updated the test.desc for this.